app/grandchallenge/reader_studies/views.py

```diff
@@ -1549,15 +1553,23 @@ def get_permission_object(self):
 
     @property
     def display_set(self):
-        return DisplaySet.objects.get(pk=self.kwargs["pk"])
+        if self.kwargs.get("pk"):
+            return DisplaySet.objects.get(pk=self.kwargs["pk"])
+
+    def get_template_names(self):
```

Comment:
Now that you can this method, the `template_name` definition above can go, right?